### DISCUSSION TITLE
about empty output on shift register

### DISCUSSION THREAD

**Discussion by benwadub** [tttapa/Control-Surface#800] [Q&A]
hi, I need to use two shift registers to use 10 leds , I ll use hc595n any advise for the wiring of the empty output? data sheet don t tell anything, do I have to let them unconnected? connect it to vcc with resistor? I can t find a serious answer on other forums, thanks for help!

**Comment by Veroledez** [comment]
Hola , no se a que te refieres ya que google me traduce a algo que no se que es, pero aqui te dejo un enlace del foro arduino en el que hay conectados 4 hc596, revisa ahi el pin que mencionas y que tienes dudas

https://forum.arduino.cc/t/cascade-of-shift-registers-74hc595-controlled-via-arduino/650524

**Comment by tttapa** [comment]
You can leave them unconnected.

**Reply by benwadub** [reply]
Thank you!
